front/css/global.css

```diff
@@ -13,5 +13,85 @@ body {
     background-color: var(--bg1);
     margin: 0;
     font-size: var(--font-size);
-    font-family: 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
+}
+
+
+/*
+    login popup css 
+*/
+.login_wrapper {
+    background-color: rgba(0,0,0,0.2);
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    width: 100%;
+    height: 100%;
+}
+
+.login_popup {
+    background-color: var(--bg3);
+    border-radius: 7px;
+    flex-direction: column;
+    padding: 10px 15px;
+    display: flex;
+    justify-content: space-evenly;
+    color: white;
+    box-shadow: 0 0 10px rgba(0,0,0,0.2);
+}
+
+.login_popup > * {
+    --margin: 10px;
+    margin-top: var(--margin);
+    margin-bottom: var(--margin);
+}
+
+.login_title {
+    width: 100%;
+    text-align: center;
+    font-size: smaller;
+}
+
+.login_label {
+    color: rgb(189, 189, 189);
+    font-size: 0.5em;
+    margin-bottom: 0;
+}
+
+.login_input {
+    margin-top: 3px;
+    outline: 0;
+    border: none;
+    border-radius: 4px;
+    padding: 5px;
+    font-size: 0.5em;
+    color: rgb(173, 173, 173);
+    background-color: var(--bg2);
+    transition: 0.1s;
+}
+
+.login_input:focus {
+    color: white;
+}
+
+.login_submit_wrapper {
+    display: flex;
+    flex-direction: row-reverse;
+}
+
+.login_submit {
+    border: none;
+    transition: 0.1s;
+    border-radius: 4px;
+    background-color: var(--accent);
+    color: white;
+    padding: 4px 8px;
+    cursor: pointer;
+}
+
+.login_submit:hover {
+    filter: brightness(0.8);
+}
+
+.login_hidden {
+    display: none;
```

Comment:
what the hell is this ?
The point of login.css or message_editor.css is to have per component css, and the point of global.css is the be a stylesheet with all rules that needs to be applied regardless of the component, so you shouldn't copy paste login.css code into global, just import login.css (and every other .css) in the html

Reply:
its even weirder because you actually import login.css

Reply:
We should keep them separate it will help us when we will switch to `.vue` files per components